src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -1023,47 +1024,90 @@ public static class Sse2
         /// __m128i _mm_set1_epi8 (char a)
         ///   HELPER
         /// </summary>
-        public static Vector128<byte> SetAllVector128(byte value) => SetAllVector128(value);
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Vector128<byte> SetAllVector128(byte value)
+        {
+            Vector128<byte> vector = Sse.StaticCast<uint, byte>(ConvertScalarToVector128UInt32(value));
+            vector = UnpackLow(vector, vector);
```

Comment:
This should be `Vector128<ushort> tmp = Sse.StaticCast<byte, ushort>(UnpackLow(vector, vector))`, because the second `UnpackLow` needs to generate `punpcklwd`

Reply:
Right, my error.